### PR TITLE
add WE subsection for MaaS_ML and MaaS_DL

### PR DESCRIPTION
**Pull request by ImenBizid**

### CODE REVIEW

File: src/docs/PML/PMLUserGuide.adoc [hunk 1]
```diff
@@ -432,18 +432,41 @@ you can also trigger the deployment of a specific VM using the Resource Manager
 
 In the following subsections, we will illustrate the life cycle of a MaaS_ML instance, from starting the generic service instance,
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:433]: the MaaS_ML instance life cycle

**Review by andrewssobral — APPROVED** [review]

File: src/docs/PML/PMLUserGuide.adoc [hunk 2]
```diff
@@ -432,18 +432,41 @@ you can also trigger the deployment of a specific VM using the Resource Manager
...
 deploying a specific model, pausing it, to deleting the instance. We will also talk about how the life cycle of a MaaS_ML instance
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:434]: We will also describe how the MaaS_ML instance life cycle can be ...

File: src/docs/PML/PMLUserGuide.adoc [hunk 3]
```diff
@@ -432,18 +432,41 @@ you can also trigger the deployment of a specific VM using the Resource Manager
...
-. <<Via Swagger UI>>
+. <<MaaS_ML Via Workflow Execution Portal>>
+. <<MaaS_ML Via Studio Portal>>
+. <<MaaS_ML Via Service Automation Portal>>
+. <<MaaS_ML Via Swagger UI>>
 
 In the description below, multiple tables represent the main variables that characterize the MaaS_ML workflows.
 In addition to the variables mentioned below, there is a set of generic variables that are common between all workflows
 which can be found in the subsection <<AI Workflows Common Variables>>.
 The management of the life cycle of MaaS_ML will be detailed in the next sub-sections.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:445]: sometimes subsection sometimes sub-section, please could u use only one form

Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:445]: same for life cycle

File: src/docs/PML/PMLUserGuide.adoc [hunk 4]
```diff
@@ -432,18 +432,41 @@ you can also trigger the deployment of a specific VM using the Resource Manager
...
 
-=== Via Studio Portal
+=== MaaS_ML Via Workflow Execution Portal
+
+Open the link:https://try.activeeon.com/automation-dashboard/#/portal/workflow-execution[Workflow Execution Portal].
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:449]: space after:

File: src/docs/PML/PMLUserGuide.adoc [hunk 5]
```diff
@@ -432,18 +432,41 @@ you can also trigger the deployment of a specific VM using the Resource Manager
...
+
+Click on the button *Submit a Job* and then search for *MaaS_ML_Service* workflow as described in the image below.
+
+image::MAAS_ML_Search.png[align=center]
+
+Check the parameters of the service and click on the button *Submit* to start a MaaS_ML Service Instance.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:455]: *Submit* button

Comment (mboussaa) [reply src/docs/PML/PMLUserGuide.adoc:455]: same, please use Service Instance everywhere or service instance

File: src/docs/PML/PMLUserGuide.adoc [hunk 6]
```diff
@@ -432,18 +432,41 @@ you can also trigger the deployment of a specific VM using the Resource Manager
...
+
+To get more information about the parameters of the service, please check the section <<Start a Generic Service Instance>>.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:457]: service parameters

File: src/docs/PML/PMLUserGuide.adoc [hunk 7]
```diff
@@ -760,14 +783,9 @@ Set the action `Finish` under Actions and click on `Execute Action`.
 
 image::MAAS_ML_Delete_Service.PNG[align=center]
 
-There are also two other actions that can be executed from Service Automation Portal which are:
-
-- *Update_MaaS_ML*: This action will update the deployed instance according to the updated variables.
-
-- *Pause_MaaS_ML*: This action will pause the service instance.
+There is also two another action that can be executed from Service Automation Portal which is:
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:786]: There are also two other actions that can be executed from the Service Automation Portal which are:

File: src/docs/PML/PMLUserGuide.adoc [hunk 8]
```diff
@@ -760,14 +783,9 @@ Set the action `Finish` under Actions and click on `Execute Action`.
...
 
-
-When running the Model Service with Singularity as an Engine, the *Pause_MaaS_ML* action can not be executed.
+- *Update_MaaS_ML_Parameters*: This action enable you to update the values of the variables associated to the enacted MaaS_ML instance according to your new preferences.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:788]: enables

Comment (mboussaa) [reply src/docs/PML/PMLUserGuide.adoc:788]: variable values

Comment (mboussaa) [reply src/docs/PML/PMLUserGuide.adoc:788]: what does mean enacted? could we use another technical word ?

File: src/docs/PML/PMLUserGuide.adoc [hunk 9]
```diff
@@ -1000,18 +1018,42 @@ you can also trigger the deployment of a specific VM using the Resource Manager
 
 In the following subsections, we will illustrate the life cycle of a MaaS_DL instance, from starting the generic service instance,
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:1019]: same please look at my previous comments

**Review by mboussaa — CHANGES_REQUESTED** [review]
Some typos to fix

File: src/docs/PML/PMLUserGuide.adoc (hunk 4 above)
Comment (ImenBizid) [reply src/docs/PML/PMLUserGuide.adoc:449]: it is an Adoc command to be able to add a link associated to Workflow Execution Portal

**Review by dianajlailaty — APPROVED** [review]

File: src/docs/PML/PMLUserGuide.adoc [hunk 10]
```diff
@@ -430,20 +430,43 @@ Once a MaaS_ML instance is up and running, it could be used for:
...
-=== Via Studio Portal
+=== MaaS_ML Via Workflow Execution Portal
+
+Open the link:https://try.activeeon.com/automation-dashboard/#/portal/workflow-execution[Workflow Execution Portal].
+
+Click on the button *Submit a Job* and then search for *MaaS_ML_Service* workflow as described in the image below.
+
+image::MAAS_ML_Search.png[align=center]
+
+Check the parameters of the service and click on the *Submit* button to start a MaaS_ML service instance.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:455]: service parameters

File: src/docs/PML/PMLUserGuide.adoc [hunk 11]
```diff
@@ -430,20 +430,43 @@ Once a MaaS_ML instance is up and running, it could be used for:
...
+
+To get more information about the parameters of the service, please check the section <<Start a Generic Service Instance>>.
+
+image::MAAS_ML_Submit.png[align=center]
+
+You can now monitor the status of the service, access its endpoint and execute its different actions :
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:461]: service status

Comment (mboussaa) [reply src/docs/PML/PMLUserGuide.adoc:461]: actions:

File: src/docs/PML/PMLUserGuide.adoc [hunk 12]
```diff
@@ -998,20 +1016,43 @@ These tasks can be easily integrated to your AI pipelines/workflows as you can s
...
+
+=== MaaS_DL Via Workflow Execution Portal
+
+Open the link:https://try.activeeon.com/automation-dashboard/#/portal/workflow-execution[Workflow Execution Portal].
+
+Click on the button *Submit a Job* and then search for *MaaS_DL_Service* workflow as described in the image below.
+
+image::MAAS_DL_Search.png[align=center]
+
+Check the parameters of the service and click on the button *Submit* to start a MaaS_DL service instance.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:1040]: service parameters

Comment (mboussaa) [reply src/docs/PML/PMLUserGuide.adoc:1040]: *Submit* button

File: src/docs/PML/PMLUserGuide.adoc [hunk 13]
```diff
@@ -998,20 +1016,43 @@ These tasks can be easily integrated to your AI pipelines/workflows as you can s
...
+
+To get more information about the service parameters, please check the section <<MaaS_DL Via Service Automation Portal>>.
+
+image::MAAS_DL_Submit.png[align=center]
+
+You can now monitor the status of the service, access its endpoint and execute its different actions :
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:1046]: service status

Comment (mboussaa) [reply src/docs/PML/PMLUserGuide.adoc:1046]: actions:

File: src/docs/PML/PMLUserGuide.adoc [hunk 14]
```diff
@@ -998,20 +1016,43 @@ These tasks can be easily integrated to your AI pipelines/workflows as you can s
...
+
+- Deploy_DL_Model : enables you to deploy a trained ML model in one click.
+- Finish_MaaS_DL : stops and deletes the service instance.
+- Redeploy_DL_Model : enables you to redeploy a DL model which was redeployed before.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:1050]: was already redeployed

File: src/docs/PML/PMLUserGuide.adoc [hunk 15]
```diff
@@ -430,20 +430,43 @@ Once a MaaS_ML instance is up and running, it could be used for:
 Using MaaS_ML, you can easily deploy and use any machine learning model as a REST Web Service on a physical or a virtual compute host on which there is an available Proactive Node. Going through the Proactive Scheduler,
 you can also trigger the deployment of a specific VM using the Resource Manager elastic policies, and eventually, deploy a Model-Service on that specific node.
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:431]: and , eventually, deploy

File: src/docs/PML/PMLUserGuide.adoc [hunk 16]
```diff
@@ -430,20 +430,43 @@ Once a MaaS_ML instance is up and running, it could be used for:
 Using MaaS_ML, you can easily deploy and use any machine learning model as a REST Web Service on a physical or a virtual compute host on which there is an available Proactive Node. Going through the Proactive Scheduler,
```
Comment (mboussaa) [inline src/docs/PML/PMLUserGuide.adoc:430]: ProActive

**Review by mboussaa — APPROVED** [review]
Approved and please take care of my latest typos

**Pull request merged**